Python/Product/Django/Intellisense/DjangoCompletionSource.cs

```diff
@@ -65,13 +65,13 @@ public DjangoCompletionSource(IGlyphService glyphService, VsProjectAnalyzer anal
             var artifactText = doc.HtmlEditorTree.ParseTree.Text.GetText(artifact.InnerRange.Start, artifact.InnerRange.Length);
             artifact.Parse(artifactText);
 
-            ITrackingSpan applicableSpan;
-            var completionSet = GetCompletionSet(session.GetOptions(_serviceProvider), _analyzer, artifact.TokenKind, artifactText, artifact.InnerRange.Start, triggerPoint, out applicableSpan);
+            var completionSet = GetCompletionSet(session.GetOptions(_serviceProvider), _analyzer, artifact.TokenKind, artifactText, artifact.InnerRange.Start, triggerPoint, out ITrackingSpan applicableSpan);
             completionSets.Add(completionSet);
         }
 
         protected override IEnumerable<DjangoBlock> GetBlocks(IEnumerable<CompletionInfo> results, SnapshotPoint triggerPoint) {
-            var doc = HtmlEditorDocument.FromTextBuffer(_buffer);
+            var buffers = _buffer.GetContributingBuffers().Where(b => b.ContentType.TypeName == TemplateHtmlContentType.ContentTypeName);
```

Comment:
@zooba , are there any rules about string comparison/equality?

Reply:
For ContentType there's an "IsOfType" or similar method that we should use rather than a string comparison.